### PR TITLE
add message in lab if project linked to organization

### LINKED ISSUE #4103: add message if project linked to organization in Project Builder/Details section
allow project collaborator to unassociate (unlink) from linked organization from within that project's Project Builder/Details section.

### PR DESCRIPTION
**Pull request by mcbouslog**
Closes #4103.

Adds message to top of project's Details section in the Project Builder if project is linked to an organization.

# Review Checklist

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Did you deploy a staging branch? - https://project-lab-show-org-link.pfe-preview.zooniverse.org/
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [x] Are the tests passing locally and on Travis?

## Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: app/pages/lab/project-details.cjsx [hunk 1]
```diff
@@ -110,6 +119,15 @@ module.exports = React.createClass
         </div>
 
         <div className="column">
+          {if @props.project.links?.organization
+            <div>
+              {if @state.organization
+                <p>This project is part of the <Link to={"/organizations/#{@state.organization.slug}"}>{@state.organization.display_name}</Link> organization.</p>
+              else
+                <p>This project is linked to <strong>Organization #{@props.project.links.organization}</strong>.</p>}
```
Comment (jelliotartz) [inline app/pages/lab/project-details.cjsx]: In the `if` condition, organization is lower case, and in the `else` condition, it's upper case - maybe good to be consistent with either upper or lower case in both?

**Comment by jelliotartz** [comment]
## PR Update

Per side conversation, Mark let me know the upper vs lower case is intentional - reasoning behind the capital O in the second conditional was that in that case it’s referring to a specific organization, and in the first we know the name and the org name is styled differently.

Tagging @beckyrother  in case she would like to have input here, but happy to approve this and move on, as it seems pretty inconsequential to me...

Examples of the two conditionals:
https://files.slack.com/files-pri/T024GDV4T-F7N1HRA01/screen_shot_2017-10-23_at_12.25.20_pm.png

https://files.slack.com/files-pri/T024GDV4T-F7N1HN30R/screen_shot_2017-10-23_at_12.24.55_pm.png

**Comment by beckyrother** [comment]
@jelliotartz Those images don't seem to work for me...

**Comment by jelliotartz** [comment]
oops, thanks @beckyrother! 

condition 1
screen shot 2017-10-23 at 12 25 20 pm

condition 2
screen shot 2017-10-23 at 12 24 55 pm

**Comment by beckyrother** [comment]
Got it, thanks! Your reasoning makes sense to me, thanks for checking!

**Comment by jelliotartz** [comment]
## PR Update

Capitalization question sorted out - LGTM, will hold on merging until after Autumnwatch initial night.

**Pull request merged**